### ISSUE TITLE
[BITV] 9.1.3.2/3.3 - Mobile (responsive) view - The visually hidden elements of the navigation on the left are always read by screen readers. That should be prevented. (8)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36926]
- [x] Fix
- [x] Server and Photos have to be updated to NC Vue v7.11.0

To be done in: 
- [x] Files
- [x] Photos
- [x] Activity
- [x] Personal / Administration Settings
- [x] Users
- [x] Apps

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-1782286125-v3-n3

</details>

- [ ] 9\.1.3.2/4.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-1782286125-v4-n2
- [ ] 9\.1.3.2/5.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-1782286125-v5-n2
- [ ] 9\.1.3.2/6.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-1782286125-v6-n2
- [ ] 9\.1.3.2/7.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-1782286125-v7-n2
- [ ] 9\.1.3.2/8.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-1782286125-v8-n2
- [ ] 9\.1.3.2/9.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-1782286125-v9-n2
- [ ] 9\.1.3.2/10.3 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-1782286125-v10-n3

**Comment by JuliaKirschenheuter** [comment]
To be done in: 
- [ ] Files
- [ ] Photos
- [ ] Activity
- [ ] Personal / Administration Settings
- [ ] Users
- [ ] Apps

**Comment by JuliaKirschenheuter** [comment]
@artonge @Pytal 

As far i've understood this task is not only about mobile view but general about hidden sidebar: as far the sidebar will be hidden - no content should be read out, right (the same is for desktop view)?

![Peek 2023-04-26 11-57](https://user-images.githubusercontent.com/6078378/234544274-de9162ef-d831-424f-baa8-9246824885bd.gif)

I would suggest to put `display: none` for parents of all navigation elements as far the navigation will be hidden and toggle it back as far the sidebar will be open again. What do you think? 

_[Accessibility concerns](https://developer.mozilla.org/en-US/docs/Web/CSS/display?retiredLocale=de#accessibility_concerns)
[display: none](https://developer.mozilla.org/en-US/docs/Web/CSS/display?retiredLocale=de#display_none)
Using a display value of none on an element will remove it from the [accessibility tree](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility#accessibility_apis). This will cause the element and all its descendant elements to no longer be announced by screen reading technology._

**Comment by Pytal** [comment]
> As far i've understood this task is not only about mobile view but general about hidden sidebar: as far the sidebar will be hidden - no content should be read out, right (the same is for desktop view)?

The one including desktop would be https://github.com/nextcloud/server/issues/36990

> I would suggest to put `display: none` or `aria-hidden="true"` for parents of all navigation elements as far the navigation will be hidden and toggle it back as far the sidebar will be open again. What do you think?

Sounds like a plan :)

**Comment by JuliaKirschenheuter** [comment]
Thanks @Pytal ,

I'm currently working on desktop version too, so i've assigned https://github.com/nextcloud/server/issues/36990 to me too. Thanks!

**Comment by artonge** [comment]
Maybe something to fix in `@nc/vue`? As most apps are using the same component.

**Comment by szaimen** [comment]
> Maybe something to fix in `@nc/vue`? As most apps are using the same component.

Makes sense since even the files app uses thst since 26 :)

**Comment by artonge** [comment]
Which Photos is using. So should be fixed, no?

**Comment by JuliaKirschenheuter** [comment]
`app-navigation-vue` used in:
- [ ] Files
- [ ] Photos
- [ ] Users
- [ ] Apps

`app-navigation` used in: (and should be fixed separately)
- [ ] Activity
- [ ] Personal / Administration Settings
